packages/plugin-core/src/test/suite-integ/RandomNoteCommand.test.ts

```diff
@@ -0,0 +1,184 @@
+import { RandomNoteConfig } from "@dendronhq/common-all";
+import { CreateNoteFactory } from "@dendronhq/common-test-utils";
+import { TestEngineUtils } from "@dendronhq/engine-test-utils";
+import * as vscode from "vscode";
+import { RandomNoteCommand } from "../../commands/RandomNote";
+import { VSCodeUtils } from "../../utils";
+import { expect } from "../testUtilsv2";
+import {
+  runLegacyMultiWorkspaceTest,
+  setupBeforeAfter,
+  withConfig,
+} from "../testUtilsV3";
+
+// common template function for RandomNoteCommand testing
+function basicTest(
```

Comment:
prefer object notation for function argument for readability. see https://wiki.dendron.so/notes/773e0b5a-510f-4c21-acf4-2d1ab3ed741e.html#prefer-using-object-notation-for-methods-ith-multiple-arguments

Reply:
Sorry, new to Typescript - when I tried to switch to object notation, the parameters can no longer be referenced within `runLegacyMultiWorkspaceTest()`.  Seems that the properties of opts aren't known in that context (Error is `Object literal may only specify known properties, and 'opts' does not exist in type 'SetupCodeConfigurationV2...`

Reply:
hmm... how did you do it? it should look like the following:
```ts
basicTest({ctx, preSetupHook, ...}: {
  ctx: ...
  preSetupHook: ...
})
```

Reply:
I see what you mean now - adjusted